lib/presenter/Chart.js

```diff
@@ -165,12 +164,14 @@ class Chart extends Events {
         var aggregateDataRange = this._aggregateDataRange;
         aggregateDataRange.removeDataSeries(nuggetView.dataSeries);
 
-        this._childElementMap.delete(nuggetView.id);
         nuggetView.remove();
```

Comment:
Before I moved this, Nugget was sometimes trying to apply throttled updates to views that should've already been deleted from the map. These calls should be synchronous.